shell/src/main/java/org/crsh/shell/impl/remoting/ServerProcess.java

```diff
@@ -65,6 +65,8 @@ public void cancel() {
       case 1:
         server.cancel(this);
         break;
+      default:
```

Comment:
If you're going to have a default case, you should perform some action, such as throwing an exception or displaying an error or at least logging (even at the debug level).